wiki/fr/fr-Privacy-Statement.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: wiki
+title: "Jamulus – Déclaration de confidentialité"
+lang: "fr"
+permalink: "/wiki/Privacy-Statement"
+---
+
+# Déclaration de confidentialité
+
+## Site web
+
+Le site web jamulus.io s'efforce de ne pas collecter vos données personnelles. Si vous consultez le site jamulus.io, votre adresse IP est envoyée à GitHub Pages. De plus, comme nous avons placé un lien vers le logo de SourceForge dans le pied de chaque page, SourceForge peut également avoir votre adresse IP.
+Nous ne plaçons pas de cookies de suivi, cependant SourceForge pourrait le faire. Si vous ne le souhaitez pas, veuillez désactiver les cookies dans votre navigateur.
+
+## Logiciel Jamulus
+
+### Utilisation des informations de profil
+
+Lorsque vous vous connectez à un serveur Jamulus public ou privé, tout ce que vous mettez dans Mon profil (dans les Paramètres) sera visible par les autres personnes sur ce serveur pendant que vous y serez connecté. Le serveur ne stocke ni n'enregistre par ailleurs les informations relatives à votre profil et l'administrateur du serveur n'y a pas accès sauf s'il y est également connecté en tant que client.
+
+Lorsque vous vous connectez à un serveur public, votre profil est également accessible aux tiers à partir du serveur central auquel ce serveur est enregistré. Il peut s'agir d'informations sur l'état du réseau public Jamulus (par exemple, [ici](http://jamulus.softins.co.uk/)), mais ne se limite pas à ça. L'information relative au profil n'est pas journalisée ou stockée par le serveur Jamulus auquel vous êtes connecté, ou par le serveur central Jamulus, mais peuvent être stockées ou traitées par des tiers.
+
+### Utilisation de l'adresse IP
+
+Lorsque vous vous connectez à un serveur public ou privé, l'administrateur du serveur peut voir votre adresse IP pendant que vous y êtes connecté. Si l'administrateur du serveur a activé la journalisation (qui est désactivée par défaut), votre adresse IP sera également journalisée et stockée dans le fichier journal du serveur.
+
+En tant qu'administrateur de serveur, lorsque vous enregistrez un serveur public auprès d'un serveur central, votre adresse IP est envoyée à CloudFlare (1.1.1.1) pour identifier votre adresse IP publique. Les adresses IP de tous les serveurs publics enregistrés auprès du serveur central peuvent également être consultées par des tiers à des fins d'information ou autres (par exemple [ici](http://jamulus.softins.co.uk/)). Votre adresse IP publique n'est par ailleurs pas journalisée ou stockée par Jamulus, mais peut être stockée ou traitée par des tiers.
+
+_Les utilisateurs concernés par l'utilisation du réseau CloudFlare peuvent [recompiler](Compiling) le code source de Jamulus afin d'utiliser une alternative pour WELL_KNOWN_HOST et WELL_KNOWN_PORT dans [global.h](https://github.com/corrados/jamulus/blob/master/src/global.h#L116)._
```

Comment:
"_https://github.com/corrados/jamulus/blob/master/src/global.h#L116_" -> "_https://github.com/corrados/jamulus/blob/master/src/global.h#L111_" (line 111, not 116)